src/action.ts

```diff
@@ -30,17 +44,65 @@ function parsePropertiesFromPayload(payload: IssuesEvent): CustomValueMap {
     Updated: properties.date(payload.issue.updated_at),
     ID: properties.number(payload.issue.id),
     Link: properties.url(payload.issue.html_url),
+    Project: properties.text(projectData?.name || ''),
+    'Project Column': properties.text(projectData?.columnName || ''),
   };
 
   return result;
 }
 
+interface ProjectData {
+  name?: string;
+  columnName?: string;
+}
+interface GetProjectDataOptions {
+  octokit: Octokit;
+  githubRepo: string;
+  issueNumber: number;
+  possible?: ProjectData;
+}
+export async function getProjectData(
+  options: GetProjectDataOptions
+): Promise<ProjectData | undefined> {
+  const {octokit, githubRepo, issueNumber, possible} = options;
+
+  const projects =
+    (
+      await octokit.rest.projects.listForRepo({
+        owner: githubRepo.split('/')[0],
+        repo: githubRepo.split('/')[1],
+      })
+    ).data || [];
```

Comment:
Does this get typed as `never`?

Reply:
No, I see it all typed correctly... Where are you getting the error?

Reply:
Marking this as resolved...